DonjinKrawler-Common/src/main/java/krawlercommon/items/ItemData.java

```diff
@@ -0,0 +1,33 @@
+package krawlercommon.items;
+
+public abstract class ItemData {
```

Comment:
Gal tikslingiau butu vadinti sia ir child klases ItemLocationData arba ItemCoordinatesData? nes siap be id tai cia kooridnates tik matau x ir y, nebent dar kazkas planuojama buti?